tests/src/test/java/com/maxio/advancedbilling/controllers/subscriptioninvoiceaccount/SubscriptionInvoiceAccountDeductServiceCreditTest.java

```diff
@@ -0,0 +1,155 @@
+package com.maxio.advancedbilling.controllers.subscriptioninvoiceaccount;
+
+import com.maxio.advancedbilling.AdvancedBillingClient;
+import com.maxio.advancedbilling.TestClient;
+import com.maxio.advancedbilling.controllers.SubscriptionInvoiceAccountController;
+import com.maxio.advancedbilling.exceptions.ApiException;
+import com.maxio.advancedbilling.models.AccountBalance;
+import com.maxio.advancedbilling.models.Customer;
+import com.maxio.advancedbilling.models.DeductServiceCredit;
+import com.maxio.advancedbilling.models.DeductServiceCreditRequest;
+import com.maxio.advancedbilling.models.IssueServiceCredit;
+import com.maxio.advancedbilling.models.IssueServiceCreditRequest;
+import com.maxio.advancedbilling.models.Product;
+import com.maxio.advancedbilling.models.ProductFamily;
+import com.maxio.advancedbilling.models.Subscription;
+import com.maxio.advancedbilling.models.containers.DeductServiceCreditAmount;
+import com.maxio.advancedbilling.models.containers.IssueServiceCreditAmount;
+import com.maxio.advancedbilling.utils.TestSetup;
+import com.maxio.advancedbilling.utils.TestTeardown;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+import org.skyscreamer.jsonassert.JSONAssert;
+import org.skyscreamer.jsonassert.JSONCompareMode;
+
+import java.io.IOException;
+import java.util.stream.Stream;
+
+import static com.maxio.advancedbilling.utils.assertions.CommonAssertions.assertNotFound;
+import static com.maxio.advancedbilling.utils.assertions.CommonAssertions.assertUnauthorized;
+import static com.maxio.advancedbilling.utils.assertions.CommonAssertions.assertUnprocessableEntity;
+import static org.assertj.core.api.Assertions.assertThat;
+
+class SubscriptionInvoiceAccountDeductServiceCreditTest {
```

Comment:
I've noticed that for this endpoint API returns 201, and in our docs its 200. Doesn't it cause any problems with the SDK?

Reply:
No, it doesn't 👍 Good catch - I'll update docs then.